soccer/gameplay/tactics/defense.py

```diff
@@ -403,6 +413,10 @@ def recalculate_threat_shot(threat_index):
         # print("Unused handlers: " + str(unused_threat_handlers))
         # print("---------------------")
 
+        # If we have nothing to block, bail
+        if not threats:
+            return
+
         smart = False
         if not smart:
```

Comment:
I mean, we already have the code block to handle smart = True, but do you know if it works / how well it works?